pkg/osquery/table/platform_tables_linux.go

```diff
@@ -49,6 +49,7 @@ func platformSpecificTables(logger log.Logger, currentOsquerydBinaryPath string)
 			allowedcmd.Lsblk, []string{"-fJp"},
 		),
 		dataflattentable.TablePluginExec(logger, "kolide_nix_upgradeable", dataflattentable.XmlType, allowedcmd.NixEnv, []string{"--query", "--installed", "-c", "--xml"}),
+		dataflattentable.TablePluginExec(logger, "kolide_wsone_uem_enroll_status", dataflattentable.JsonType, allowedcmd.Ws1HubUtil, []string{"status", "--enroll"}),
```

Comment:
I wonder if it's worth vetting this command with the customer first. I'm especially curious if we should do it with or without `--enroll`

Reply:
(I pinged the customer)

Reply:
I can pull out the `--enroll`, but as I haven't seen the output, I'm not sure if the status itself returns data in JSON. The `--enroll`, `--dependency`, and `--profile` do confirm they are JSON formatted.

Reply:
There's also `--app`, which isn't described in the documentation, but is an option shown, so I'm not sure how that data would return either.

Reply:
I got some back from customer. Might be interesting to do each of:
* /opt/vmware/ws1-hub/bin/ws1HubUtil status --enroll
* /opt/vmware/ws1-hub/bin/ws1HubUtil status --dependency
* /opt/vmware/ws1-hub/bin/ws1HubUtil status --profile